plugins/database/mssql/mssql.go

```diff
@@ -69,12 +76,37 @@ func (m *MSSQL) Initialize(ctx context.Context, req dbplugin.InitializeRequest)
 	if err != nil {
 		return dbplugin.InitializeResponse{}, err
 	}
+
+	usernameTemplate := getString(req.Config, "username_template")
```

Comment:
We can use `strutil.GetString` from your other PR. 🙂

Reply:
:+1: that was the plan!